src/main/js/utils/Schedule.js

```diff
@@ -11,13 +11,33 @@ class Schedule extends Set {
       super(data);
       this.gap = data.gap;
       this.credits = data.credits;
+      this.gpa = 0.00;
```

Comment:
The problem with having this be a private field is if you calculate the gpa and then add another course then the old gpa would be wrong.

Reply:
Removed.